transports/msmq/sender-side-distribution_select-destination_core_[6].partial.md

```diff
@@ -0,0 +1 @@
+In Version 6.2 and above it is possible to override the virtual method `SelectDestination`. The method provides access to the `DistributionStrategyContext` that enables implementing more advanced distribution scenarios, such as distributing based on the headers of the message. When `SelectDestination` is overridden, do not call `base.SelectDestination` since the base method calls `SelectReceiver` for backward compatibility reasons. `SelectReceiver` can throw a `NotImplementedException`.
```

Comment:
> Version 6.2 and above

to

> Versions 6.2 and above

and isnt this partial set to only show in v6?

> `SelectReceiver` can throw a `NotImplementedException`.

"can"? under what circumstances?

Reply:
yeah not sure why this is mentioned. I think we use the NotImplementedException in when overriding SelectDestination to mark SelectReceiver as a "dummy implementation" as it won't be called, but this isn't expressed well by this sentence. I removed it.